lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -446,7 +445,12 @@ namespace Js
             // Clear disable implicit call bit as initialization code doesn't have any side effect
             Js::ImplicitCallFlags saveImplicitCallFlags = scriptContext->GetThreadContext()->GetImplicitCallFlags();
             scriptContext->GetThreadContext()->ClearDisableImplicitFlags();
-            JavascriptFunction::CallRootFunctionInScript(function, Js::Arguments(callInfo, args));
+
+            {
+                AutoSuspendDebuggerRecording autoSuspendDebuggerRecording(scriptContext);
```

Comment:
>autoSuspendDebuggerRecording [](start = 45, length = 28)

Instead of AutoSuspendDebuggerRecording you can rename AutoProfilingUserCode and use that instead. The purpose of AutoProfilingUserCode is same (both profiler and debugger should be not tracking internal library code)